testcases/change_pasword/change_password.robot

```diff
@@ -0,0 +1,84 @@
+    ...    ${register_email}    ${default_user_data.password}    ${default_user_data.password}
+    Click Register Button
+    Click Continue Button After Registered
+    Click On Login Link
+    Login Successfully With    ${register_email}    ${default_user_data.password}
+    Click On My Account Link
+    Click On Change Password Link
+    Input Old And New Password Then Click Change Password Button  
+    ...    ${default_user_data.password}    ${password_data.changed_01}    ${password_data.changed_01}
+    Verify Change Password Success Message
+    Close Change Password Success Message
+    Wait Until Change Password Success Message Disappears
+    Click On Logout Link
+    Click On Login Link
+    Login Successfully With    ${register_email}    ${password_data.changed_01}
+
+Unable to change password twice in a row
+    [Documentation]    User is redirected to Login page when trying to change password twice in a row
+    ${register_email}=    Generate Random Email Based On DateTime
+    Input Required Data Into Register Form    ${default_user_data.firstName}    ${default_user_data.lastName}  
+    ...    ${register_email}    ${default_user_data.password}    ${default_user_data.password}
+    Click Register Button
+    Click Continue Button After Registered
+    Click On Login Link
+    Login With    ${register_email}    ${default_user_data.password}
+    Verify Log Out Link Appears
+    Click On My Account Link
+    Click On Change Password Link
+    Input Old And New Password Then Click Change Password Button  
+    ...    ${default_user_data.password}    ${password_data.changed_01}    ${password_data.changed_01}
+    Verify Change Password Success Message
+    Input Old And New Password Then Click Change Password Button  
+    ...    ${password_data.changed_01}    ${password_data.changed_02}    ${password_data.changed_02}
+    Verify Page Title of Login Page
+
+Unable to change back password to the 3 recent used
+    [Documentation]    User is not able to change password back to last 3 used passwords
+    ${register_email}=    Generate Random Email Based On DateTime
+    Input Required Data Into Register Form    ${default_user_data.firstName}    ${default_user_data.lastName}  
+    ...    ${register_email}    ${default_user_data.password}    ${default_user_data.password}
+    Click Register Button
+    Click Continue Button After Registered
+    Login And Change Password Successful    ${register_email}    ${default_user_data.password}    ${password_data.changed_01}
+    Click On Logout Link
+    Login And Change Password Successful    ${register_email}    ${password_data.changed_01}    ${password_data.changed_02}
+    Click On Logout Link
+    Login And Change Password Successful    ${register_email}    ${password_data.changed_02}    ${password_data.changed_03}
+    Click On Logout Link
+    Login And Change Password Successful    ${register_email}    ${password_data.changed_03}    ${password_data.changed_04}
+    Click On Logout Link
```

Comment:
It's okay for now, but in future projects, let's try to set preconditions using APIs instead of the UI. This makes scripts run faster and stable